### ISSUE TITLE
Security, improvement

### ISSUE THREAD

**Issue by AquAn48** [TeamWin/android_bootable_recovery#18]
Good day to all.
Make the option-so that the sdcard directory can be (optionally) protected with a password - so that it is impossible to view / copy the contents of the sdcard directory without a password. This is useful when you lose your phone or put it in for repair - so that outsiders can't read confidential information (Bank cards, etc.). At the moment - although Android can be protected with a password - TWRP is a window into the smartphone not only for the owner of the device but also for dishonest people.

     (I apologize for my English - I use machine translation).

**Comment by mauronofrio** [comment]
First thing, use english.
Then how the TWRP can manage something that system doesn't manage? If you encrypt the sdcard and then the system is unable to manage it, how did you will be able use sdcard in system?

**Comment by AquAn48** [comment]
if you need access to the sdcard, the smartphone owner enters a password and manages the data, but someone else can't.
and to block access was optional - only if desired, for example in the advanced settings.
I apologize for my English - I use machine translation.

**Comment by mauronofrio** [comment]
Yes and then people extract the sdcard from your phone and use it in an other device

**Comment by AquAn48** [comment]
this refers to the internal sdcard (non-removable)
![IMG_20200512_203253](https://user-images.githubusercontent.com/65136616/81726393-f92e0b80-948f-11ea-9df2-4a133382e838.jpg)
